docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -347,13 +280,15 @@
             (get_distributed_cloud_role.stdout is not defined) or
             get_distributed_cloud_role.stdout != "systemcontroller"
 
+          # TODO(wasnio): delete this once dm pod can handle missing webhooks
```

Comment:
Not required, but one future work item may be: update the helmchart to collaborate with the pod and the webhook